index.js

```diff
@@ -105,23 +113,86 @@ function initApp(callback) {
     /*
      * configure routes
      */
-    app.post('/api/v1/substitution', (req, res) => {
-      if (!req.isAuthenticated() || req.user.level < config.user.level.substitute) {
-        res.status(401).send('{"error":"not authenticated or not allowed"}');
-        return;
-      }
+var Compendium = require('./lib/model/compendium');
+
+    // get all compendia for list in authorView.html
+    app.get('/api/v1/substitution/all', (req, res) => {
```

Comment:
That is not clear from the codebase, and no argument for putting this code in here even now.

Even with the new local data path, the endpoint provided by muncher lists the compendia.